src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2729,8 +2730,9 @@ private bool IsMisplacedModifier(SyntaxListBuilder modifiers, SyntaxList<Attribu
                     return true;
                 }
 
-                var incompleteMember = _syntaxFactory.IncompleteMember(attributes, modifiers.ToList(), type.IsMissing ? null : type);
-                if (incompleteMember.ContainsDiagnostics)
+                var incompleteMember = _syntaxFactory.FieldDeclaration(attributes, modifiers.ToList(), CreateVariableDeclarationWithTypeAndSingleMissingDeclarator(type), CreateMissingSemicolonToken());
+                //var incompleteMember = _syntaxFactory.IncompleteMember(attributes, modifiers.ToList(), type.IsMissing ? null : type);
```

Comment:
>//var incompleteMember = _syntaxFactory.IncompleteMember(attributes, modifiers.ToList(), type.IsMissing ? null : type); [](http://example.com/codeflow?start=16&length=119)

In general, we do not submit obsolete commented out code. If you want to clarify something for the reader, consider putting that in words.